merlin/examples/dev_workflows/restart.yaml

```diff
@@ -0,0 +1,46 @@
+description:
+  name: restart
+  description: A simple ensemble for testing step restarts.
+
+batch:
+  type: local
+
+env:
+  variables:
+    OUTPUT_PATH: ./studies
+    N_SAMPLES: 10
+
+study:
+- name: restart_and_sample_step
+  description: runs 
+  run:
+    cmd: |
+      echo "sample: $(V1)"
+      exit $(MERLIN_RESTART)
+    max_retries: 3
+
+- name: indiv_followup_step
+  description: step that can start immedeatly after a sample has finished
+  run:
+      cmd: echo "followup complete :)"
+      depends: [restart_and_sample_step]
+
+
+- name: bottleneck_step
+  description: step that can only start upon complete of all samples in previous step
+  run:
+      cmd: echo "bottleneck complete :)"
+      depends: [restart_and_sample_step_*]
```

Comment:
should we add a step that happens after the indiv_followup all complete? This will help us test chords of chains

Reply:
Something like this?
```
- name: indiv_followup_step_2
  description: step that can start immediately after a followup has finished
  run:
      cmd: echo "$(V1) followup 2 complete :)"
      depends: [indiv_followup_step]
```

Reply:
Maybe I don't understand how to make a chord. I thought it was just having `_*` after a dependency.